### PR TITLE
ci: set global timeout

### PR DESCRIPTION
**Pull request by FeelyChau**

### CODE REVIEW

File: run_tests.ts [hunk 1]
```diff
@@ -12,9 +12,11 @@ const indexSpec = process.argv.indexOf('--spec');
 if (indexSpec !== -1 && process.argv[indexSpec + 1]) {
   spec = [ process.argv[indexSpec + 1] ];
 }
+
 const runner = new JasmineCtor();
+runner.jasmine.DEFAULT_TIMEOUT_INTERVAL = 20 * 6000;
```
Comment (yorkie) [inline run_tests.ts:17]: 20 seconds?

File: run_tests.ts [hunk 2]
```diff
@@ -12,9 +12,11 @@ const indexSpec = process.argv.indexOf('--spec');
...
 runner.loadConfig({
-  spec_files: spec || [ 'src/**/*_test.ts', 'test/**/*.ts' ],
+  spec_files: spec || [ 'src/**/*_test.ts', 'test/**/job.ts' ],
```
Comment (yorkie) [inline run_tests.ts:19]: Why using the job instead of *?

Comment (FeelyChau) [reply run_tests.ts:19]: It's test code, need to be removed.

File: run_tests.ts (hunk 1 above)
Comment (FeelyChau) [reply run_tests.ts]: Modified to 20 mins.

**Comment by FeelyChau** [comment]
@yorkie  Ready.

**Pull request merged**